File: platforms/ios/Dezel/Dezel/Modules/Form/JavaScriptFormModule.swift
/**
 * @class JavaScriptFormModule
 * @super JavaScriptModule
 * @since 0.7.0
 */
open class JavaScriptFormModule: JavaScriptModule {

	//--------------------------------------------------------------------------
	// MARK: Methods
	//--------------------------------------------------------------------------

	/**
	 * @method configure
	 * @since 0.7.0
	 */
	override open func configure(context: JavaScriptContext) {
		context.registerClass("dezel.form.TextInput", with: JavaScriptTextInput.self)
		context.registerClass("dezel.form.TextArea", with: JavaScriptTextArea.self)
	}
}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/modules/form/JavaScriptFormModule.kt
package ca.logaritm.dezel.modules.form

import ca.logaritm.dezel.core.JavaScriptContext
import ca.logaritm.dezel.core.JavaScriptModule

/**
 * @class JavaScriptFormModule
 * @super JavaScriptModule
 * @since 0.1.0
 */
open class JavaScriptFormModule: JavaScriptModule() {

	/**
	 * @method configure
	 * @since 0.7.0
	 */
	override fun configure(context: JavaScriptContext) {
		context.registerClass("dezel.form.TextInput", JavaScriptTextInput::class.java)
		context.registerClass("dezel.form.TextArea", JavaScriptTextArea::class.java)
	}
}